libraries/ESP8266WiFi/src/WiFiServer.h

```diff
@@ -100,6 +100,8 @@ class WiFiServer {
   uint16_t port() const;
   void close();
   void stop();
+  void end();
+  virtual operator bool();
```

Comment:
`explicit` and `const`?

This is implicit conversion, and c++ (also implicit) defaults for operator declaration have many repercussions
https://en.cppreference.com/w/cpp/language/implicit_conversion

`explicit` restricts this to the `if (obj)` construct mentioned in the Arduino reference
https://www.arduino.cc/reference/en/libraries/ethernet/ifserver/

Reply:
yes. `explicit` is better.
strange `Serial.println(server);` stopped working while `Serial.println(!server);` works.